src/components/languageMenu/LanguageMenuPure.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import OutsideClickHandler from 'react-outside-click-handler'
+import LanguageContext from 'src/contexts/language'
+import './language-menu.less'
+import configPages from 'src/config'
+
+const LanguageMenuPure = ({
+  closeLanguageMenu,
+  toggleLanguageMenu,
+  isLanguageMenuShown
+}) => {
+  const locationPathname = window.location.href
+    .split('/')
+    .filter(function(element) {
+      return element !== ''
+    })
+    .pop()
+  if (
+    configPages[locationPathname] &&
+    configPages[locationPathname].languages !== -1
```

Comment:
Could you move long lines outside of if statement and name it accordingly what it does

Reply:
done